server/constants/platform.ts

```diff
@@ -4,19 +4,30 @@ import moment from 'moment';
 import { SupportedCurrency } from './currencies';
 
 const isProdOrStaging = config.env === 'production' || config.env === 'staging';
-const MIGRATION_DATE = moment('2024-10-01T00:00:00Z');
+
+/**
+ * Defines the transition date and time from OCI to Ofico.
+ */
+export const PLATFORM_MIGRATION_DATE = moment('2024-10-01T00:00:00Z');
 
 /**
  * Returns the platform constants based on a function to check if the platform has been migrated.
  */
 const getPlatformConstants = (checkIfMigrated: () => boolean) => ({
+  get OCICollectiveId() {
+    return 8686;
+  },
+
+  get OFTCollectiveId() {
+    return 845576;
+  },
+
   get PlatformCollectiveId() {
     if (checkIfMigrated()) {
-      // ofico
-      return 835523;
+      return this.OFTCollectiveId;
+    } else {
+      return this.OCICollectiveId;
```

Comment:
Added that to make it easier to selectively fetch profiles from https://github.com/opencollective/opencollective-api/pull/10338